spread.yaml

```diff
@@ -164,7 +164,13 @@ backends:
             - centos-8-64:
                   workers: 6
                   storage: preserve-size
-                  image: centos-stream-8
+                  image: centos-8-64
+            - centos-9-64:
+                  workers: 6
+                  storage: preserve-size
+                  image: centos-9-64
+                  manual: true
```

Comment:
Do we need to keep this manual?

Reply:
I'll remove the manual as it is not included in the workflow, so far it is failing to run
I'll add a following pr to enable centos-9, but for that we need to have the xdelta3 dependency available